src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,40 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS_IN_STORAGE = 10;
+    private final K[] keyArray = (K[]) new Object[MAX_ELEMENTS_IN_STORAGE];
+    private final V[] valueArray = (V[]) new Object[MAX_ELEMENTS_IN_STORAGE];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class fields have default values 
int default value is 0 - no need to assign it